api/models/bigquery_jobs.py

```diff
@@ -0,0 +1,14 @@
+from api import app, db
+
+
+class BigqueryJobs(db.Model):
+    __tablename__ = "bigquery_jobs"
+
+    table_name = db.Column(db.String(100), primary_key=True, nullable=False)
+    index = db.Column(db.String(100), server_default="0")
+    bigquery_table_name = db.Column(db.String(100))
+    type = db.Column(db.String(100))
+    last_updated_at = db.Column(
+        db.DateTime,
+        server_default=app.config["DEFAULT_LAST_UPDATED_TIME_FOR_DATA_MIGRATION"],
```

Comment:
Instead of getting this data from config, can we define it here in this class an utilize it? I don't think we are using it in any other place, so getting it from config doesn't make much sense. Thoughts?

Reply:
Agree, we are not using it in any other place. We can add this value to the same class.